client/controllers/feeds.coffee

```diff
@@ -0,0 +1,19 @@
+Feeds = require '/imports/collections/feeds.coffee'
+
+Template.feeds.onCreated ->
+  Meteor.subscribe 'feeds'
```

Comment:
Using `@subscribe` would be better so the subscription is stopped when the template is destroyed.